src/replica/replica_stub.cpp

```diff
@@ -493,10 +509,13 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
     std::deque<task_ptr> load_tasks;
     uint64_t start_time = dsn_now_ms();
     for (auto &dir : dir_list) {
-        if (dir.length() >= 4 &&
-            (dir.substr(dir.length() - 4) == ".err" || dir.substr(dir.length() - 4) == ".gar" ||
-             dir.substr(dir.length() - 4) == ".bak")) {
-            ddebug("ignore dir %s", dir.c_str());
+        if (dir.length() < 4) {
+            continue;
+        }
```

Comment:
When dir length is valid, it should be added into `load_tasks`, why do you remove it?

Reply:
Oh, it's error